.github/docs/qemu.md

```diff
@@ -87,7 +94,33 @@ Then restart libvirt
 sudo systemctl restart libvirtd
 ```
 
-## Misc
+## Troubleshooting
+
+### VMs are not properly cleaned up after a failed `constellation create` command
+
+Terraform may fail to remove your VMs, in which case you need to do so manually.
 
 - List all domains: `virsh list --all`
-- Destroy domain with nvram: `virsh undefine --nvram <name>`
+- Destroy domains with nvram: `virsh undefine --nvram <name>`
+
+### VMs have no internet access
+
+`iptables` rules may prevent your VMs form properly accessing the internet.
+Make sure your rules are not dropping forwarded packages.
+
+List and check your rules:
+
+```shell
+$ sudo iptables -S
+-P INPUT ACCEPT
+-P FORWARD DROP
+-P OUTPUT ACCEPT
+-N DOCKER
```

Comment:
A bit confusing that you only need to execute the first line and the rest is the output, but I blame that more on `iptables` itself.